src/bin/CMakeLists.txt

```diff
@@ -21,6 +21,30 @@ target_link_libraries(
     hyriseBenchmarkLib
 )
 
+# Configure playground for self driving database / group 01
+add_executable(
+    hyrisePlaygroundGroup01
```

Comment:
I don‘t think that we should include the playground. @Bensk1 - if you consider this worth keeping, maybe we can move it to `hyrise-archive`?